orchestrator/android/ExoPlayer-Java/app/src/main/java/io/streamroot/ctl/delivery/client/samples/orchestrator/exoplayer/ExoPlayerQosModule.java

```diff
@@ -0,0 +1,50 @@
+package io.streamroot.ctl.delivery.client.samples.orchestrator.exoplayer;
+
+import com.google.android.exoplayer2.ExoPlaybackException;
+import com.google.android.exoplayer2.ExoPlayer;
+import com.google.android.exoplayer2.Player;
+import com.google.android.exoplayer2.Player.EventListener;
+import com.google.android.exoplayer2.source.TrackGroupArray;
+import com.google.android.exoplayer2.trackselection.TrackSelectionArray;
+
+import io.streamroot.ctl.delivery.client.core.QosInterfaceBase;
+import io.streamroot.ctl.delivery.client.core.VideoPlaybackStatePublic;
```

Comment:
Do we need to add `Public` to `VideoPlaybackStatePublic`, or we were having a collision with another internal class

Reply:
I found it more clean internally, but from the outside, there shouldn't be any collision no, do you want me to make the changes ?